drivers/usbdev/composite.h

```diff
@@ -95,7 +95,7 @@
 #  define COMPOSITE_REMOTEWAKEUP      (0)
 #endif
 
-#define NUM_DEVICES_TO_HANDLE         (4)
+#define NUM_DEVICES_TO_HANDLE         (8)
```

Comment:
shouldn't we make `NUM_DEVICES_TO_HANDLE` configurable with Kconfig ?
then we can reuse this value in the board initialization logic, like here https://github.com/apache/nuttx/blob/856526adee25cdb9987ff800767a71ec9059d1ea/boards/arm/stm32h7/nucleo-h743zi/src/stm32_composite.c#L178 which is now not possible as `NUM_DEVICES_TO_HANDLE` is not available outside `drivers/usbdev`

Reply:
Hi, raiden00pl, this is a good idea, i will submit a new PR for this change.